modules/core/src/main/scala/sup/data/Tagged.scala

```diff
@@ -16,7 +16,9 @@ object Tagged {
     * The only place where it should be passed to is [[sup.HealthReporter.fromChecks]],
     * for determining the status of the wrapping check. In other cases, it's probably useless, as it discards the tag completely.
     */
-  implicit def catsReducibleForTagged[Tag]: Reducible[Tagged[Tag, *]] = Reducibles.by(λ[Tagged[Tag, *] ~> Id](_.health))
+  implicit def catsReducibleForTagged[Tag]: Reducible[Tagged[Tag, *]] = Reducibles.by(new (Tagged[Tag, *] ~> Id) {
```

Comment:
`FunctionK` lambda syntax is not supported on Scala 3, hence the more verbose rewrite.